<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser API for Acer Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Open With URL</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Content</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEditURL">
             <property name="text">
              <string>http://www.google.com</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QRadioButton" name="radioBtnSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radioBtnUrl">
             <property name="text">
              <string>GoToURL</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QRadioButton" name="radioBtnTab">
             <property name="text">
              <string>SelectTab</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer">
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushBtnOpen">
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Current Tab</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushBtnUpdateCurrentTab">
             <property name="text">
              <string>UpeateInfo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEditCurrentIndex">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushBtnGetIndex">
           <property name="text">
            <string>Get ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Close Tab</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Index             </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBoxTabIndex"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushBtnClose">
               <property name="text">
                <string>Close</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>DB Reading</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TabCount:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Index:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4" colspan="2">
             <widget class="QSpinBox" name="spinBoxDBIndex"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelTabCount">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="6">
             <layout class="QFormLayout" name="formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>URL</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditDbUrl"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Title</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditTitle"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>ThumbnailPath</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditThumbnailPath"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Tab ID</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEditTabID"/>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushRefreshCount">
              <property name="text">
               <string>Refresh Count</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Signal Log</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QTextEdit" name="textEditSignalLog">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushBtnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnCloseTabClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnOpen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnShowBrowserClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnGetIndex</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnGetCurrentTabIndexClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnUpdateCurrentTab</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnUpdateCurrentTabClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRefreshCount</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnRefreshDBClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDBIndex</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSpinBoxValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnCloseTabClicked()</slot>
  <slot>onBtnShowBrowserClicked()</slot>
  <slot>onBtnUpdateCurrentTabClicked()</slot>
  <slot>onBtnGetCurrentTabIndexClicked()</slot>
  <slot>onBtnRefreshDBClicked()</slot>
  <slot>onSpinBoxValueChanged(int)</slot>
 </slots>
</ui>
